cfspopcon/formulas/beta.py

```diff
@@ -4,9 +4,7 @@
 from ..unit_handling import Quantity, Unitfull, convert_units, ureg
 
 
-def _calc_beta_general(
-    average_electron_density: Unitfull, average_electron_temp: Unitfull, average_ion_temp: Unitfull, magnetic_field: Unitfull
-) -> Unitfull:
+def _calc_beta_general(average_total_pressure: Unitfull, magnetic_field: Unitfull) -> Unitfull:
     """Calculate the average ratio of the plasma pressure to the magnetic pressure due to a magnetic_field.
 
     Using equation 11.58 from Freidberg, "Plasma Physics and Fusion Energy" :cite:`freidberg_plasma_2007`
```

Comment:
Citation only really says `2*m_0 * <p> / B**2` so I'd say citation is still ok. and we only change how we calculate the average pressure. What do you think?

Reply:
I agree!